test/Microsoft.DocAsCode.MarkdownRewriters.Tests/AzureMarkdownRewritersTest.cs

```diff
@@ -1148,5 +1150,57 @@ public void TestAzureMarkdownRewriters_NestedList_Bug551102()
             var result = AzureMarked.Markup(source);
             Assert.Equal(expected.Replace("\r\n", "\n"), result);
         }
+
+        #endregion
+
+        #region Azure migration marked
+
+        [Fact]
+        [Trait("Related", "AzureMarkdownMigrationRewriters")]
+        public void TestAzureMarkdownMigrationRewriters_AzureVideoLink()
+        {
+            var azureVideoInfoMapping =
+                new Dictionary<string, AzureVideoInfo>{
+                    {
+                        "azure-ad--introduction-to-dynamic-memberships-for-groups",
+                        new AzureVideoInfo
+                        {
+                            Id = "azure-ad--introduction-to-dynamic-memberships-for-groups",
+                            Link = "https://channel9.msdn.com/Series/Azure-Active-Directory-Videos-Demos/Azure-AD--Introduction-to-Dynamic-Memberships-for-Groups/player/"
+                        }
+                    }
+                };
+
+            var source = @"> [AZURE.VIDEO azure-ad--introduction-to-dynamic-memberships-for-groups]";
+            var expected = @"> [AZURE.VIDEO azure-ad--introduction-to-dynamic-memberships-for-groups]";
+
+            var result = AzureMigrationMarked.Markup(source, "sourceFile.md", null, azureVideoInfoMapping);
+            Assert.Equal(expected.Replace("\r\n", "\n"), result);
+        }
+
+        [Fact]
+        [Trait("Related", "AzureMarkdownMigrationRewriters")]
+        public void TestAzureMarkdownMigrationRewriters_AzureVideoLinkNoMapping()
+        {
+            var azureVideoInfoMapping =
+                new Dictionary<string, AzureVideoInfo>{
+                    {
+                        "fake-azure-ad--introduction-to-dynamic-memberships-for-groups",
+                        new AzureVideoInfo
+                        {
+                            Id = "fake-azure-ad--introduction-to-dynamic-memberships-for-groups",
+                            Link = "https://channel9.msdn.com/Series/Azure-Active-Directory-Videos-Demos/Azure-AD--Introduction-to-Dynamic-Memberships-for-Groups/player/"
+                        }
+                    }
+                };
+
+            var source = @"> [AZURE.VIDEO azure-ad--introduction-to-dynamic-memberships-for-groups]";
+            var expected = @"> [AZURE.VIDEO azure-ad--introduction-to-dynamic-memberships-for-groups]";
```

Comment:
I've already created the video syntax, do you forget that?

---
In reply to: [73271562](https://github.com/dotnet/docfx/pull/559#discussion_r73271562) [](ancestors = 73271562)